taskfiles/rpk.yml

```diff
@@ -22,3 +22,10 @@ tasks:
            -X ${ver_pkg}.rev={{.SHORT_SHA}} \
            -X ${cont_pkg}.tag={{.RPK_VERSION}}" \
         -o "{{.BUILD_ROOT}}/go/$GOOS/bin" ./...
+  tidy:
+    desc: mod tidy rpk
+    env:
+      CGO_ENABLED: "0"
+    dir: "{{.SRC_DIR}}/src/go"
+    cmds:
+    - sh -c 'for dir in $(find . -maxdepth 1 -type d ! -path .); do cd $dir; go mod tidy; cd ..; done'
```

Comment:
nit: this can use YAML multiline syntax to avoid invoking another shell, for example:

```suggestion
    cmds:
    - |
      for dir in $(find . -maxdepth 1 -type d ! -path .); do
        cd $dir && go mod tidy && cd ..
      done
```

Reply:
Nice! I've squashed this into the commit introducing this task. Turns out I could do this on one line as well without dropping into shell, I think I originally had a trailing `'`, but multiline looks nicer / is more understandable.